indico/modules/events/surveys/templates/management/survey.html

```diff
@@ -1,6 +1,38 @@
 {% extends 'events/surveys/management/_survey_base.html' %}
 
 {% block content %}
+    {% if survey.private and survey.state.name in ('active_and_clean', 'active_and_answered') %}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-lock"></div>
+                <div class="text">
+                    <div class="label">
+                        {%- trans %}The survey is private{% endtrans -%}
+                    </div>
+                    {% trans -%}
+                        The survey is accessible only by people with this
+                    {%- endtrans %}
+                    <a href="{{ url_for('.display_survey_form', survey, token=survey.uuid, _external=True) }}"
```

Comment:
I don't really understand how it works, what should be in that dictionary that I would return form the locator ? in that example there is `return dict(self.registration_form.locator, token=self.uuid)` but what would be the first argument in my case ?